src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -15,6 +16,8 @@ export enum MultiSelectType {
     Summarized = 'Summarized',
 }
 
+export type MultiSelectSize = 's' | 'm';
```

Comment:
There has been a whole discussion on enum vs union types, you can check it here: https://weare.frontify.com/document/42#/frontend-react/typescript/the-enum-vs-union-types-discussion
All 'size' types in all components use enum so I think it's better to be consistent and change it to enum.